website/content/docs/enterprise/lease-count-quotas.mdx

```diff
@@ -25,17 +25,23 @@ the lease counters will be shared, regardless of which node in the Vault cluster
 receives lease generation requests. Lease quotas can be imposed across Vault's API,
 or scoped down to API pertaining to specific namespaces or specific mounts.
 
-A quota that is defined in the `root` namespace is inherited by all namespaces
-and mounts, essentially to the entire Vault API.
-
-Lease count quotas defined on a namespace will take precedence over the inherited
-quotas. Lease count quotas defined for a mount will take precedence over inherited
-and namespace quotas. The limits on the namespace and mount quotas can either be
-increased or decreased. If the inherited quota is very restrictive and if it is
-desired to relax the limits in one namespace, or on a specific mount, it can be
-done using this precedence model. On the other hand, if the inherited quota is
-very liberal and if it is desired to further restrict usages in a specific
-namespace or mount, that can be done using the precedence model too.
+A quota that is defined in the `root` namespace with no specified path is inherited by all namespaces.
+Essentially, it applies to the entire Vault API unless a more specific quota has been defined
+for a specific API path.
+
+Lease count quotas defined on a namespace will take precedence over the global
```

Comment:
```suggestion
Lease count quotas defined on a namespace takes precedence over the global
```

Reply:
It might be better to modify the paragraph by putting the content in an unordered list since "...will take precedence..." is repetitive here.